tests/test_netbox.py

```diff
@@ -29,7 +29,29 @@ def get_inv(requests_mock, plugin, pagination, version, **kwargs):
                     json=json.load(f),
                     headers={"Content-type": "application/json"},
                 )
-    return plugin().load()
+    return plugin(include_vms=include_vms).load()
+
+
+def get_vm_inv(requests_mock, plugin, pagination, version, **kwargs):
```

Comment:
This function basically duplicates the code in get_inv().
Maybe we need to extract setting up the mocks into a different function and  call it from get_inv(). 

For example
```
def _create_mock(requests.mock, pagination, version, application, resource, **kwargs):
    """initialises mock objects for testcase"""
    if not pagination:
        with open(f"{BASE_PATH}/mocked/{version}/{resource}.json", "r") as f:
            requests_mock.get(
                "http://localhost:8080/api/{application}/{resource}/?limit=0",
                json=json.load(f),
                headers={"Content-type": "application/json"},
            )
    else:
        for offset in range(3):
            with open(f"{BASE_PATH}/mocked/{version}/{resource}-{offset}.json", "r") as f:
                url = "http://localhost:8080/api/{application}/{resource}/?limit=0"
                requests_mock.get(
                    f"{url}&offset={offset}" if offset else url,
                    json=json.load(f),
                    headers={"Content-type": "application/json"}
                )
```
```
def get_inv(requests_mock, plugin, pagination, version, include_vms=False, **kwargs):

    _create_mock(requests.mock, pagination, version, "dcim", "devices")
    if include_vms:
        _create_mock(requests_mock, pagination, version, "virtualization", "virtual-machines")

    return plugin(include_vms=include_vms).load()
```

Reply:
probably also means you will have to rename the data mock files to "virtual-machines".

Reply:
Dropped that in and was working with just a few tweaks.